src-executables/Main-trident.hs

```diff
@@ -73,14 +77,19 @@ optParserInfo :: OP.ParserInfo Options
 optParserInfo = OP.info (OP.helper <*> versionOption <*> optParser) (
     OP.briefDesc <>
     OP.progDesc "trident is a management and analysis tool for Poseidon packages. \
-                \More information: \
-                \https://poseidon-framework.github.io. \
-                \Report issues: \
+                \Report issues here: \
                 \https://github.com/poseidon-framework/poseidon-hs/issues"
     )
 
 versionOption :: OP.Parser (a -> a)
-versionOption = OP.infoOption (showVersion version) (OP.long "version" <> OP.help "Show version")
+versionOption = OP.infoOption (showVersion version) (OP.long "version" <> OP.help "Show version number")
```

Comment:
I thought not, because `--version` should imho return the software version in a simple, machine-readable format. It should be easily possible to parse the output in a pipeline.

Reply:
OK, that makes sense. Good.